crates/ruff_wasm/README.md

```diff
@@ -0,0 +1,51 @@
+# Ruff WASM
+
+### ⚠️ WARNING: This API is experimental and may change at any time
```

Comment:
```suggestion
> [!WARNING]
>
> This API is experimental and may change at any time
```

I'd suggest this instead of using a header:

<img width="729" alt="Screenshot 2024-07-15 at 13 27 30" src="https://github.com/user-attachments/assets/97155bfe-d70d-49c7-95a9-ab47b090070b">

Reply:
I tried this and it didn't seem to work on npm, only on github
![image](https://github.com/user-attachments/assets/b476c42f-6709-4a02-8fc1-65e8fc011b2a)

Reply:
Oh, I didn't realize this was going to be rendered on NPM. It's fine then to either use a header or block quote:

```
> **⚠️ WARNING: This API is experimental and may change at any time**
```
> **⚠️ WARNING: This API is experimental and may change at any time**